content/api/relay-webhooks.apib

```diff
@@ -8,30 +8,29 @@ Relay Webhooks are a way to instruct SparkPost to accept inbound email on your b
 
 By configuring a relay webhook for a specified inbound domain, those inbound messages can be forwarded to a specified target over HTTP.  Before you create a relay webhook, be sure to first create an inbound domain that is properly configured. To create an inbound domain for your account, please use our Inbound Domains API. The Relay Webhooks API provides the means to create, list, retrieve, update, and delete a relay webhook.
```

Comment:
Relay webhooks forward emails sent to an inbound domain to a specified target over HTTP. Before you create a relay webhook, be sure to first create and configure inbound domain (link)